extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -182,10 +184,10 @@ public function content( $params ){
 			}
 		}
 
-		$this->thumbWidth = self::IMG_WIDTH;
-		$this->thumbHeight = self::IMG_HEIGHT;
-		$this->thumbRows = self::THUMBS_NUM_ROWS;
-		$this->numThumbsPerRow = self::THUMBS_PER_ROW;
+		$this->thumbWidth = static::IMG_WIDTH;
+		$this->thumbHeight = static::IMG_HEIGHT;
+		$this->thumbRows = static::THUMBS_NUM_ROWS;
+		$this->numThumbsPerRow = static::THUMBS_PER_ROW;
 		$this->trendingArticles = $trendingArticles;
 		$this->mainPageUrl = Title::newMainPage()->getFullUrl();
 		//$this->larrSrc = $this->wg->ExtensionsPath."/wikia/AppPromoLanding/images/arrow-left-long.svg";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)